common/persistence/sql/sqlplugin/postgres/execution.go

```diff
@@ -274,10 +284,15 @@ func (pdb *db) SelectFromTransferTasks(ctx context.Context, filter *sqlplugin.Tr
 
 // DeleteFromTransferTasks deletes one or more rows from transfer_tasks table
 func (pdb *db) DeleteFromTransferTasks(ctx context.Context, filter *sqlplugin.TransferTasksFilter) (sql.Result, error) {
-	if filter.MinTaskID != nil {
-		return pdb.conn.ExecContext(ctx, rangeDeleteTransferTaskQuery, filter.ShardID, *filter.MinTaskID, *filter.MaxTaskID)
```

Comment:
was this intentional delete?
it does look like we have the wrong filter here if this condition is true though...